### ISSUE TITLE
Z Stepper overheating causing prints to fail

### ISSUE THREAD

**Issue by v1ncen72** [spinixguy/Sovol-SV06-firmware#9]
I used the setting from the printer.cfg file seen here, and for big prints (20 hours+) , I have failure after about 18 hours due to overheat of z stepper motor.

I have now changed the config to:
[tmc2209 stepper_z]
uart_pin: PA15
run_current: 1.000
stealthchop_threshold: 0
interpolate: False
sense_resistor: 0.150
uart_address: 3
diag_pin: PA7

(which I found in https://github.com/bassamanator/Sovol-SV06-firmware/blob/master/printer.cfg)

Is it the right thing to do? Should I lower the run_current?

Any advice appreciated.

(recently bought this SV06 and just built a case to enclose it)
![image (2)](https://user-images.githubusercontent.com/27919721/232059046-30bf88ff-49a5-4d5a-a7c4-8b0f5ccc51e2.png)

Also, to troubleshoot this further, is there any way to monitor this temperature in Mainsail anywhere?

**Comment by spinixguy** [comment]
It is entirely up to you as to what you do and change. My values came from SOVOL. 

What you might want to check is the voltage of your motors. Its stamped on the bottom of the motor. 
The early batch of printers have 0.8 motors. Later ones had 1.3. Mine are 1.3 and they do not overheat. However, it wont hurt anything to lower the values as long as you do not go too low.  

If you have the 1.3 then my guess is its an environment thing. z steppers should not be working that hard even on long time prints. X and Y do all the work.

**Comment by v1ncen72** [comment]
Thanks for answering so quickly!

I think I am unlucky.. and got one from earlier batch :(
I just did a 3-hour print and the Z motors are burning, Can't even touch them for more than a few seconds :(

![IMG-1347](https://user-images.githubusercontent.com/27919721/232118958-79851866-eae7-411c-8721-967618938d1a.jpg)

**Comment by spinixguy** [comment]
yeah i would lower the values.

**Comment by eXsoR65** [comment]
> It is entirely up to you as to what you do and change. My values came from SOVOL. 
> 
> 
> 
> What you might want to check is the voltage of your motors. Its stamped on the bottom of the motor. 
> 
> The early batch of printers have 0.8 motors. Later ones had 1.3. Mine are 1.3 and they do not overheat. However, it wont hurt anything to lower the values as long as you do not go too low.  
> 
> 
> 
> If you have the 1.3 then my guess is its an environment thing. z steppers should not be working that hard even on long time prints. X and Y do all the work.


Non-related to the issue, but I've been trying to confirm if I have the "later batch" stepper motors, that run at 1.3a, as they are not they don't state the Amps on it. 

All I see on the steppers are the brand and model. I've looked everywhere to not find any info. @spinixguy can you confirm if that the case for you? 

Brand: Shengyang Motor
Model: 42BYGH1634-A-20Db
